### PR TITLE
feat: added ConstructorBlocks to module for external using

### PR DESCRIPTION
**Pull request by VitaliiDC8**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ export * from './context/theme';
 export * from './context/analyticsContext';
 
 export * from './containers/PageConstructor';
+export * from './containers/PageConstructor/components/ConstructorBlocks';
```
Comment (gorgeousvlad) [inline src/index.ts:6]: lets add index.ts file to `PageConstructor/components` with content `export {ConstructorBlocks} from './ConstructorBlocks';`

and then in `src/containers/PageConstructor/index.ts` add line `export * from './components';`

**Comment by NikitaCG** [comment]
I think we need release this feature with https://github.com/gravity-ui/page-constructor/pull/463
@VitaliiDC8 @gorgeousvlad

**Review by NikitaCG — APPROVED** [review]

File: src/containers/PageConstructor/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+export * from './components';
```
Comment (gorgeousvlad) [inline src/containers/PageConstructor/index.ts:1]: please add it in the end of file

Comment (VitaliiDC8) [reply src/containers/PageConstructor/index.ts]: moved

**Review by gorgeousvlad — APPROVED** [review]

**Pull request merged**